src/kit.ts

```diff
@@ -1186,7 +1238,11 @@ export async function readKitsFile(filePath: string, workspaceFolder?: string, e
         }
         return [];
     }
-    const kits = kits_raw as Kit[];
+    const kits = (kits_raw as Kit[]).map(kit => {
+        // Serialized kits are trusted for backwards compatibility if not otherwise specified.
```

Comment:
Is this because if they're serialized we're assuming they've already been ran? Or why is this assumption being made?

Reply:
CMake Tools has a cache of previously discovered kits. If we don't trust this cache, then all previously discovered kits will be considered untrusted until kits are scanned again, which potentially breaks user workflows.

Reply:
Makes sense.